packages/aws-cdk-lib/aws-stepfunctions/lib/states/map.ts

```diff
@@ -63,12 +63,28 @@ export interface MapProps {
   readonly itemsPath?: string;
 
   /**
-   * The JSON that you want to override your default iteration input
+   * The JSON that you want to override your default iteration input (mutually exclusive  with `itemSelector`).
+   *
+   * @deprecated Step Functions has deprecated the `parameters` field in favor of
+   * the new `itemSelector` field
+   *
+   * @see
+   * https://docs.aws.amazon.com/step-functions/latest/dg/input-output-itemselector.html
    *
    * @default $
    */
   readonly parameters?: { [key: string]: any };
 
+  /**
+   * The JSON that you want to override your default iteration input (mutually exclusive  with `parameters`).
+   *
+   * @see
+   * https://docs.aws.amazon.com/step-functions/latest/dg/input-output-itemselector.html
+   *
+   * @default $
```

Comment:
Does $ have a significance as a default? Can this docstring be descriptive?

Reply:
`$` represents the entire state input in Step Functions. Because this is essentially a replacement of `parameters`, I kept the docstring the same as what it was for `parameters`.